packages/renderer/src/lib/ui/Link.svelte

```diff
@@ -19,7 +20,11 @@ onMount(() => {
 
 function click() {
   if (href) {
-    window.openExternal(href);
+    if (href.startsWith('/')) {
+      router.goto(href);
+    } else {
+      window.openExternal(href);
+    }
```

Comment:
I think here we may have a problem, a relative link could be `../foo` or `./bar`

so I don't know if we should proceed like that or we have 'internal-link' vs 'external-link' so when referencing a link we do know what we want to do/achieve

Reply:
I'm not sure where we use any relative links like that, but what do you think about switching it around (see new commit)? It checks for `startsWith('http')`, which I think should be a safer test.

Reply:
I don't think we need to be smart here.

I'm pretty sure we will have links to `podman-desktop://` etc

so I would give the responsibility to the author using `<Link>` component so it's when using the component that we tell open it externally or open it locally

Else we'll always have bugs around this part

Reply:
1. onClick
1. internalRef
1. externalRef

then consumer choose how he wants to use that component

Reply:
Ok, component updated to use internalRef/externalRef props in new commit.

Reply:
thanks, we are less smart but more robust